### ISSUE TITLE
WebpDrawable.setLoopCount(LOOP_INTRINSIC) uses frameCount instead loopCount

### ISSUE THREAD

**Issue by WindSekirun** [zjupure/GlideWebpDecoder#55]
* Device: Samsung A8 (2018) 7.0 , Samsung S10+ 10.0 ... almost devices
* Sample Project: https://github.com/WindSekirun/Webpdecodetest
* Glide / GlideWebpDecoder version: 4.10.0 / 1.6.4.10.0

Hello,

I'm trying to play a specific animation webp image only once, and I got a file that set loop_count to 1.

The problem is that when i play it with Glide, the image repeats infinitely. So I tried a little trace.

1. In #38, GlideWebpDecoder supports **obtaining original webp loop count**. 
2. But, `WebpDrawable.getLoopCount()` will return -1 until we called 'WebpDrawable.setLoopCount()`.
3. If, We call `WebpDrawable.setLoopCount(LOOP_INTRINSIC)`, it will call `state.frameLoader.getLoopCount()` -> `webpDecoder.getTotalIterationCount();`.
4. But, `WebpDecoder.getTotalIterationCount()` return `mWebPImage.getFrameCount() + 1` instead of `mWebPImage.getLoopCount()`. [Code](https://github.com/zjupure/GlideWebpDecoder/blob/master/webp_decoder/src/main/java/com/bumptech/glide/integration/webp/decoder/WebpDecoder.java#L172)

So, It seems to work differently from the actual behavior.

If `WebpDecoder.getTotalIterationCount()` returns `netscapeLoopCount` instead of `frameCount + 1`, this seems to fix the problem. (I confirmed the operation through reflection.)

```
private fun getNetscapeLoopCount(drawable: WebpDrawable): Int {
        val stateField = drawable.javaClass.getDeclaredField("state").apply { isAccessible = true }
        val state = stateField.get(drawable)

        val loaderField = state.javaClass.getDeclaredField("frameLoader").apply { isAccessible = true }
        val loader = loaderField.get(state) as WebpFrameLoader

        val decoderField = loader.javaClass.getDeclaredField("webpDecoder").apply { isAccessible = true }
        val decoder = decoderField.get(loader) as WebpDecoder

        return decoder.netscapeLoopCount
    }
```

Below is the related log (webp0.webp is infinity image, webp1.webp is once image)

```
D/WebpDecodeTest: imageUrl https://windsekirun.github.io/Webpdecodetest/webp0.webp loopCount = -1
D/WebpDecodeTest: imageUrl https://windsekirun.github.io/Webpdecodetest/webp0.webp netscapeloopCount = 0
```

```
D/WebpDecodeTest: imageUrl https://windsekirun.github.io/Webpdecodetest/webp1.webp loopCount = -1
D/WebpDecodeTest: imageUrl https://windsekirun.github.io/Webpdecodetest/webp1.webp netscapeloopCount = 1
```

Thanks.

**Comment by WindSekirun** [comment]
I made PR for this, #56.

**Comment by zjupure** [comment]
thanks for your pr and looks good! I will merge it

**Comment by WindSekirun** [comment]
@zjupure when will it be released?

**Comment by zjupure** [comment]
@WindSekirun library 1.6.1 has published to jcenter for glide 4.10.0 and 4.11.0
